lib/wibox/widget/slider.lua

```diff
@@ -228,7 +232,10 @@ local slider = {mt={}}
 -- @tparam[opt=gears.shape.rectangle] gears.shape shape
 -- @see gears.shape
 
---- The handle grab cursor.
+--- The cursor icon while grabbing the handle.
+-- The availabe cursor names are:
```

Comment:
typo

Reply:
(and same word on other lines)

Reply:
Thanks for the review. The typo is now fixed.